src/main/webapp/gameBoard.html

```diff
@@ -40,8 +40,10 @@
             <h5 id="headings"> Game Name </h5>
             </br>
  
-            <div id="leaderboard-container">
-                LeaderBoard Elements will be populated Here
+            <div id="leaderboard-container" onload = "loadUsers()">
+                <body onload="loadUsers()">
+                    <ol id="leader-board"></ol>
+                </body>
```

Comment:
It looks like you are calling "loadUsers()" twice in a row, did you mean to do this?

Reply:
^ removed the body tag and call